// thirdweb/dist/cjs/react/web/ui/ConnectWallet/screens/Buy/swap/StepConnector.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.StepConnectorArrow = StepConnectorArrow;
const jsx_runtime_1 = require("react/jsx-runtime");
const CustomThemeProvider_js_1 = require("../../../../../../core/design-system/CustomThemeProvider.js");
const basic_js_1 = require("../../../../components/basic.js");
function StepConnectorArrow(props) {
    const theme = (0, CustomThemeProvider_js_1.useCustomTheme)();
    return ((0, jsx_runtime_1.jsx)(basic_js_1.Container, { flex: "row", center: "both", style: {
            width: "100%",
            height: "12px",
            position: "relative",
            marginTop: "-1px",
            zIndex: 1000,
        }, children: (0, jsx_runtime_1.jsxs)("svg", { role: "presentation", width: "32", height: "16", viewBox: "0 0 32 16", fill: "none", style: {
                display: "block",
            }, children: [(0, jsx_runtime_1.jsx)("path", { d: "M1 0L16 15L31 0", fill: theme.colors.tertiaryBg, stroke: props.active ? theme.colors.accentText : theme.colors.borderColor, strokeWidth: "1", strokeLinecap: "square", strokeLinejoin: "miter" }), (0, jsx_runtime_1.jsx)("path", { d: "M8 0L16 7.5L24 0", fill: "none", stroke: props.active
                        ? theme.colors.accentText
                        : theme.colors.secondaryIconColor, strokeWidth: "1", strokeLinecap: "square", strokeLinejoin: "miter" })] }) }));
}
